Sources/SwiftSyntaxBuilder/ResultBuilderExtensions.swift

```diff
@@ -12,52 +12,92 @@
 
 import SwiftSyntax
 
-extension CodeBlockItemListBuilder {
-  public static func buildExpression(_ expression: some ExprSyntaxProtocol) -> Component {
-    return buildExpression(CodeBlockItemSyntax(item: .expr(ExprSyntax(expression))))
+public extension CodeBlockItemListBuilder {
+  static func buildExpression(_ expression: some ExprSyntaxProtocol) -> Component {
+    buildExpression(CodeBlockItemSyntax(item: .expr(ExprSyntax(expression))))
```

Comment:
I prefer to keep the `public` on the functions. Putting `public` on the extension makes it non-obvious that the functions inside of it are public and you can easily change one of these functions and accidentally change the public API.